backend-project/small_eod/letters/tests/test_views.py

```diff
@@ -112,3 +120,36 @@ def get_update_data(self):
     def validate_update_item(self, item):
         self.assertEqual(item["id"], self.obj.pk)
         self.assertEqual(item["comment"], f"{self.obj.comment}-updated")
+
+    def create_test_list(self, url, field, reverse=False):
+        param = "-" + field if reverse else field
+        response_ordered = self.client.get(url, {"ordering": param})
+        if "__" in field:
```

Comment:
Ja bym zamiast rozwiązywać relacje ręcznie bym pomyślał jak tego uniknąć.
Jeżeli zapiszesz:
`
model.objects.filter.order_by(*map(to_camel_case,field)).values('id', flat=True)
`
to masz z czym porównać i zweryfikować, a nie musisz rozwiązywać relacji.
Jednak jak użyjesz parameterized to możesz na sztywno wprowadzić oczekiwany kod, a wtedy jeszcze prościej będzie to prześledzić.